pyClient/commands/zeth_token_approve.py

```diff
@@ -7,7 +7,7 @@
 from click import command, argument, option, pass_context, ClickException, Context
 
 
-@command(name="token_approve")
+@command(name="token-approve")
```

Comment:
Thanks for this. It's definitely inconsistent, but we may want to just remove the `name` arg. We seem to be able to use `token-approve` AND `token_approve` on the command line, independently of what value is here, and when it's removed, the help text shows `token-approve`.  Also, the other commands do not specify `name` explicitly.

For complete consistency, we can switch from `token_approve` to `token-approve` in the test script `scripts/test_zeth_cli_token`.  Thanks.

Reply:
I've gone with your suggestion of removing the line completely. Also fixed the reference in the script file. Thanks!